test/helpers/gather-os-metrics.spec.js

```diff
@@ -0,0 +1,20 @@
+const chai = require('chai');
+
+chai.should();
+
+const gatherOsMetrics = require('../../helpers/gather-os-metrics');
+
+describe('helpers', () => {
+  describe('gather-os-metrics', () => {
+    describe('when invoked', () => {
+      it('then ...', () => {
+        const span = { os: [], responses: [] };
+
+        gatherOsMetrics({}, span);
+
+        // todo: not sure what should I test, maybe the resulted span structure?
+        // todo: also this component has got some internal timing events?
```

Comment:
Nope but it's async. I'd recommend injecting the process pid in the function so that you can change it while testing

https://github.com/soyuka/pidusage/blob/master/lib/stats.js#L44 code is really readable and well commented, if you have any question just ask me ;)